referrer-policy/no-referrer-policy/no-referrer-policy.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Check that subresource gets the referrer URL when no explicit
+           Referrer Policy is set.</title>
+    <script src='/resources/testharness.js'></script>
+    <script src='/resources/testharnessreport.js'></script>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body>
+    <h1>Check that subresource gets the referrer URL when no explicit Referrer
+        Policy is set.</h1>
+
+    <p>This page loads the subresource and waits for the message containing the
+       referrer URL from the child.</p>
+
+    <script>
+
+      // Display this page's location.
+      appendPageInfo("Parent frame");
+
+      // Embed the iframe on a different domain - load subresource.
+      var iframe = document.createElement('iframe');
+      iframe.type = 'text/javascript';
+      iframe.src = "http://www1." + location.hostname + ":" + location.port +
+                   "/referrer-policy/no-referrer-policy" +
+                   "/no-referrer-policy.subresource.html";
+      iframe.width="100%";
+      iframe.height="200px";
+      document.body.appendChild(iframe);
+
+      var referrerTest = async_test(
+          "Checks that referer is passed to the subresource (iframe).");
+
+      function iFrameListener(event) {
```

Comment:
I have refactored this so that each iframe receives and ID as a query parameter. The iframe then sends back that ID along with the referrer. I then reuse that ID to run the appropriate test scenario. Also added some sanity checks.